pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -40,8 +40,19 @@ from timezones cimport (
 _zero_time = datetime_time(0, 0)
 _no_input = object()
 
+
 # ----------------------------------------------------------------------
 
+def int_op_deprecated(obj):
```

Comment:
ok this looks good. can you rename to 
``maybe_integer_op_deprecated`

Reply:
sure